cerberus-web/src/main/java/com/nike/cerberus/service/SafeDepositBoxService.java

```diff
@@ -636,6 +647,7 @@ protected SafeDepositBoxV2 convertSafeDepositBoxV1ToV2(SafeDepositBoxV1 safeDepo
                         .withIamPrincipalArn(
                             String.format(
                                 DomainConstants.AWS_IAM_ROLE_ARN_TEMPLATE,
+                                "aws",
```

Comment:
This is also part of a deprecated API. Same reasoning as above.

> There is a chance. Right now it's only there for backwards compatibility. Maybe we can use aws/aws-cn depending on the feature flag or adding an extra field in the V1 KMS controller. But since the KMS is deprecated, I'd avoid putting too much working into it.